pkgs/servers/mpd/i386.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/decoder/plugins/FfmpegDecoderPlugin.cxx b/src/decoder/plugins/FfmpegDecoderPlugin.cxx
+index 6986453..167fc07 100644
+--- a/src/decoder/plugins/FfmpegDecoderPlugin.cxx
++++ b/src/decoder/plugins/FfmpegDecoderPlugin.cxx
+@@ -20,8 +20,8 @@
+ /* necessary because libavutil/common.h uses UINT64_C */
+ #define __STDC_CONSTANT_MACROS
+ 
+-#include "lib/ffmpeg/Time.hxx"
+ #include "config.h"
++#include "lib/ffmpeg/Time.hxx"
```

Comment:
I’m kind of done with `mpd` by now, I just stream stuff to a pulseaudio sink. No idea why I thought setting one up again was a good idea. Certainly a waste of time …